Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM debian:jessie
```

Comment:
You're right. It should be Centos7. But let's stick with this official Docker image for Go, because we don't have one in Centos.